src/tox.rs

```diff
@@ -24,15 +24,49 @@ pub enum Answer {
     Event(Event),
 }
 
+struct Friend {
+    number: u32,
+    public_key: String,
+    name: String,
+    status: rstox::core::UserStatus,
+    status_message: String,
+    last_online: u64,
+}
+
 fn run_request(tox: &mut rstox::core::Tox, request: &Request) -> Option<Response> {
     use Request as R;
 
     match request {
         R::Info => {
-            let name = tox.get_name();
             let tox_id = format!("{}", tox.get_address());
+
+            let name = tox.get_name();
+            let status = tox.get_status();
+            let status_message = tox.get_status_message();
+
+            let friends: Vec<_> = tox.get_friend_list()
```

Comment:
but I don't see how you use `friends` variable after initialization

Reply:
I need to update ws-tox-protocol dependency but I have no working machine. `Friend` structure also should be used from `ws-tox-protocol`.